srtp/srtp.c

```diff
@@ -1532,6 +1532,110 @@ srtp_get_session_keys(srtp_stream_ctx_t *stream, uint8_t* hdr,
   return NULL;
 }
 
+static srtp_err_status_t
+srtp_estimate_index(srtp_rdbx_t *rdbx,
```

Comment:
Took me a while to understand this function, some comments would have been nice. The NO_64BIT_MATH does not make it easier. In the end I think this function is correct at least for the case when there is 64bit maths.